src/js/tech/html5.js

```diff
@@ -1245,7 +1245,17 @@ Html5.disposeMediaElement = function(el) {
     // wrapping in an iife so it's not deoptimized (#1060#discussion_r10324473)
     (function() {
       try {
-        el.load();
+        if (el.paused) {
+          el.load();
+        } else {
+          const playPromise = el.play();
+
+          if (playPromise) {
+            playPromise.then(() => el.load());
```

Comment:
we probably want to add an empty function (or use silencePromise https://github.com/videojs/video.js/blob/master/src/js/utils/promise.js#L24) to the second part of then to make sure that it doesn't throw another error

either
```js
playPromise.then(() => el.load(), () => {});
```
or
```
// and add an import for silencePromise
silencePromise(playPromise.then(() => el.load()));
```

Reply:
and below as well

Reply:
Added in `reset`